uber/site_sections/budget.py

```diff
@@ -46,3 +46,157 @@ def mpoints(self, session):
         all = [(sum(mpu.amount for mpu in mpus), group, mpus)
                for group, mpus in groups.items()]
         return {'all': sorted(all, reverse=True)}
+
+    @ajax
+    def add_promo_code_words(self, session, text='', part_of_speech=0):
```

Comment:
The specific use case that I believe caused Rob's eyes to light up was "FlamingCyberSword"